docs/runtimes/images.md

```diff
@@ -24,16 +24,22 @@ All images are published to
 The images are:
 
 - `mlrun/mlrun`: An MLRun image includes preinstalled OpenMPI and other ML packages. Useful as a base image for simple jobs. 
+- `mlrun/mlrun-kfp`: The same as `mlrun/mlrun` with the addition of KFP 1.8 already installed. **This image includes KFP.**
 - `mlrun/mlrun-gpu`: The same as `mlrun/mlrun` but for GPUs, including Open MPI. 
 - `mlrun/ml-base`: Image for file acquisition, compression, Dask jobs, simple training jobs and other utilities.
-- `mlrun/jupyter`: An image with Jupyter giving a playground to use MLRun in the open source. Built on top of jupyter/scipy-notebook, with the addition of MLRun and several demos and examples.
+- `mlrun/jupyter`: An image with Jupyter giving a playground to use MLRun in the open source. Built on top of jupyter/scipy-notebook, with the addition of MLRun and several demos and examples. 
+- `mlrun/mlrun-api`: The server's image.
+- `mlrun/log-collector`: A service that collects the logs for all runs and stores them in a persistent storage
 
-```{admonition} Note
+```{admonition} Notes
 When using the `mlrun` or `mlrun-gpu` image, use PyTorch versions up to and including than 2.0.1, but not higher. 
 You can build your own images with newer CUDA for later releases of PyTorch.
 ```
+### Install with or without KFP?
 
+MLRun uses KFP 1.8. If you want to use packages whose dependencies conflict with KFP 1.8, such as pydantic 2, then use any image except `mlrun/mlrun-kfp`. You can then install the package in your environment and your images without affecting the MLRun functionality.  
```

Comment:
```suggestion
MLRun uses KFP 1.8. If you want to use packages whose dependencies conflict with KFP 1.8, such as pydantic 2, then use any image except `mlrun/mlrun-kfp`. You can then install the package in your environment and your images without affecting the MLRun functionality.  
```

`mlrun-kfp` image is used to compile the workflow (& perform some mlrun agent-y operations like sending notifications, etc). the workflow functions may run based on `mlrun/mlrun` image which comes with pydantic v2.

Reply:
@liranbg Pls give a few more details. You compile a workflow with one image, and the functions use another image? Can you give an example?